app/javascript/src/components/Dashboard/Contacts/Table/Column.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Avatar, Typography, Dropdown } from "neetoui";
+
+export const Column = [
```

Comment:
Prefer to name this as `buildContactTableColumnData`.

Refer: https://www.loom.com/share/67cd6e3f3fab4498a062c9ca0d2ffa3f

Reply:
I actually tried adding this to utils.js first and got this error. That's why I added it to Column.jsx

<img width="987" alt="Screenshot 2022-09-16 at 2 15 41 PM" src="https://user-images.githubusercontent.com/45137335/190596837-60270e68-56aa-443a-b06f-ee395d1b7e26.png">

Reply:
On second thoughts, I think we can keep this rendering logic within the `constants` file itself given that we don't have args for the util function. We'd only need to rename the file as `constants.jsx`, that is with a `jsx` extension. That's fine. Because having a `jsx` extension doesn't necessarily assert that it's a React component. Having a PascalCase filename along with a `jsx` extension would be the only case where we can consider the file as a React component. Lemme know if that sounds good.